scss/_pagination.scss

```diff
@@ -154,7 +154,7 @@
   &:first-child .page-link {
     margin-right: var(--#{$prefix}pagination-margin-x-first-last);
 
-    &:not(:empty) {
+    &:not(:empty) :not([data-bs-a11y]) {
```

Comment:
I still need to figure what is the best way to target it without using something that users will not use 🤔